server/app/Models/Doc.php

```diff
@@ -230,67 +220,45 @@ public function getPagesAttribute()
         return $this->getPages();
     }
 
-    public function getCommentCount()
-    {
-        return $this->comments()->count();
-    }
-
     public function getCommentCountAttribute()
     {
-        return $this->getCommentCount();
+        // note that this is only the top level comment count
+        return $this->comments
+            ->filter(function ($item) {
+                return !$item->isNote();
+            })
+            ->count()
+            ;
```

Comment:
I can replace this with a self-join and some stuff on the query itself, but it won't read as well. Was trying to avoid raw SQL and messiness until we know we need it. Who knows, in a month the whole idea of differentiating a count of "comments" and "notes" might be done away with!

I'm imagining our current "busy" workload looks something like 10 people interacting on a document with maybe 200 top-level comments/notes (the only kind we work with in this method). That's ~2,000 (these counts trigger on every page load, so kinda depends how much people reload a document) things to iterate through, most of which are already going to be loaded from the database due to other needs (like responding with the comment content), so it doesn't seem that inefficient to me. I'll do a little testing though.